homeassistant/components/fritz/diagnostics.py

```diff
@@ -22,6 +22,7 @@ async def async_get_config_entry_diagnostics(
         "entry": async_redact_data(entry.as_dict(), TO_REDACT),
         "device_info": {
             "model": avm_wrapper.model,
+            "unique_id": avm_wrapper.unique_id,
```

Comment:
this might be treated as sensitive data, since the `unique_id` is just the MAC of the device 🤔 
Would suggest to shorten it to the first 4 bytes - so to say, redact the last 2 bytes from the MAC like `xx:xx:xx:xx**REDACTED**`